### PR TITLE
Add parser for "ua" tld with tests

### PR DESCRIPTION
**Pull request by DarkMristov**

### CODE REVIEW

File: parser_ua.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+		NameServers:    regexp.MustCompile(`nserver: *(.*)`),
+		DomainStatus:   regexp.MustCompile(`(?s)nserver:.*?status: *(.*?)\ncreated`),
+		CreatedDate:    regexp.MustCompile(`domain:(?:.*\s)*?created: *(.*)`),
+		UpdatedDate:    regexp.MustCompile(`domain:(?:.*\s)*?modified: *(.*)`),
+		ExpirationDate: regexp.MustCompile(`domain:(?:.*\s)*?expires: *(.*)`),
+		Emails:         regexp.MustCompile(`abuse-email: *(.*)`),
+		RegistrarName:  regexp.MustCompile(`registrar: *(.*)`),
+	},
+	techRegex: &RegistrantRegex{
+		// (?:[^n][^\\][^a]|[^n][^\\]a|[^n]\\a|[^n]\\[^a]|n\\[^a]|n[^\\][^a]) - negative lookahead alternative
```
Comment (icamys) [inline parser_ua.go]: Is this comment necessary?

Comment (DarkMristov) [reply parser_ua.go]: Only for explanation of this regex functional.

Comment (icamys) [reply parser_ua.go]: If something requires any explanation please create variable with corresponding clear name. Comments should explain only your intentions.

Comment (DarkMristov) [reply parser_ua.go]: Done

**Pull request merged**